dxp/how_clay_arrives_to_dxp.md

```diff
@@ -0,0 +1,223 @@
+
+Of course you can import it directly into your theme even if you didn't use the generator to create it, but in this section, we are going to see how it works on the default ones.
+
+---
+
+Let's say we created a theme named `my-theme`, we used the generator and we setted `unstyled` as a base.
+
+Since `unstyled` is the most basic theme of the chain, you'll have a simple build procress:
+
+`clay > unstyled > my-theme`
+
+But, this procress will change depending on the base you've chosen:
+
+| Base     | Process                                         |
+| -------- | ----------------------------------------------- |
+| unstyled | `clay > unstyled > my-theme`                    |
+| styled   | `clay > unstyled > styled > my-theme`           |
+| classic  | `clay > unstyled > styled > classic > my-theme` |
+| admin    | `clay > unstyled > styled > admin > my-theme`   |
+
+Keep in mind that each base has it's properties and are defined for specific scopes
+
+|                                                           | unstyled | styled | classic | admin |
+| --------------------------------------------------------- | -------- | ------ | ------- | ----- |
+| Clay                                                      | ✅       | ✅     | ✅      | ✅    |
+| Bourbon (deprecated)                                      | ✅       | ✅     | ✅      | ✅    |
+| Mixins (deprecated)                                       | ✅       | ✅     | ✅      | ✅    |
+| Legacy Liferay components (now in Clay)                   |          | ✅     | ✅      | ✅    |
+| Common Liferay components (Portlets)                      |          | ✅     | ✅      | ✅    |
+| Liferay customizations of Clay components                 |          | ✅     | ✅      | ✅    |
+| Compat Layer: compatibility layer for Bootstrap 2 and 3   |          | ✅     | ✅      | ✅    |
+| Main CSS of the Liferay websites and pages                |          |        | ✅      |       |
+| CSS Custom Properties Layer: Modern customization of Clay |          |        | ✅      |       |
+| Hello World Widget: Front Page Banner                     |          |        | ✅      |       |
+| Main CSS of the Liferay administration panels             |          |        |         | ✅    |
+
+From a more technical point of view, setting the base through the generator will create a setting named `liferayTheme` in the `package.json` file of the theme folder.
+
+```json
+{
+	"name": "my-liferay-theme",
+	"version": "1.0.0",
+	"main": "package.json",
+	"keywords": ["liferay-theme"],
+	"liferayTheme": {
+		"baseTheme": "styled",
+		"templateLanguage": "ftl",
+		"version": "7.0"
+	},
+	"devDependencies": {
+		"gulp": "^3.8.10",
+		"liferay-theme-deps-7.1": "*",
+		"liferay-theme-tasks": "*"
+	}
+}
+```
+
+[Source](https://github.com/liferay/liferay-js-themes-toolkit/tree/master/packages/liferay-theme-tasks#liferaytheme)
+
+## How to import Clay into a custom-theme?
```

Comment:
This section looks more like a _how to_ than an specification on how things are done currently. If we are using this pattern in our code, I would like to see a link to an example in `liferay-portal`, for example. If not, we should perhaps move it to another _how to_ document.